applications/prompt-keda-lsstcam/values-usdfprod-prompt-processing.yaml

```diff
@@ -72,7 +72,7 @@ prompt-keda:
   s3:
     imageBucket: rubin-summit
     endpointUrl: https://sdfembs3.sdf.slac.stanford.edu
-    cred_file_auth: True
+    aws_profile: embargo
```

Comment:
I wonder if it might be worth keeping `cred_file_auth` here just because we really do need it, whether or not it's the default. I assume `aws_profile` is only useful if `cred_file_auth` is set?

Reply:
> aws_profile is only useful if cred_file_auth is set
Yes. 

I'm adding a check for that.  The deployment would fail if the parameters aren't consistent.  

With that, maybe it's okay not keeping `cred_file_auth` there?